java/vector/src/main/codegen/templates/BaseWriter.java

```diff
@@ -90,6 +90,7 @@
     void clear();
     void copyReader(FieldReader reader);
     MapWriter rootAsMap();
+    MapWriter rootAsMap(Boolean caseSensitive);
```

Comment:
I would prefer to configure the case sensitivity directly on ComplexWriter as well. But I have only found MapWriters lowering-case of field names which confused me a bit and make me thinking the case sensitivity probably only applies to the MapWriters.